dev-qt/libqt4pas/libqt4pas-2.5.0.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Pascal Qt interface"
+DESCRIPTION="Pascal wrapper around Qt interfaces.
+Just the necessary stuff for Lazarus is wrapped here."
+HOMEPAGE="http://users.telenet.be/Jan.Van.hijfte/qtforfpc/fpcqt4.html"
+SOURCE_URI="http://users.telenet.be/Jan.Van.hijfte/qtforfpc/V2.5/qt4pas-V2.5_Qt4.5.3.tar.gz"
+CHECKSUM_SHA256="825423db80da4df5c21816c0392b3394cddfe2f3293dfd08ace84941726affea"
+SOURCE_DIR="qt4pas-V2.5_Qt4.5.3"
+LICENSE="GNU LGPL v2.1"
+COPYRIGHT="2005-2011 Jan Van hijfte"
+REVISION="1"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libqt4pas$secondaryArchSuffix = $portVersion
+	lib:libqt4pas$secondaryArchSuffix = 5.2.5 compat >= 5.2
+	"
+	
```

Comment:
No need for an empty line here, between PROVIDES and REQUIRES.
BTW, the right order is:
```
PROVIDES
REQUIRES

PROVIDES_devel
REQUIRES_devel

BUILD_REQUIRES
BUILD_PREREQUIRES
```